### PR TITLE
Rename games on tvOS

### PR DESCRIPTION
**Pull request by leolobato**
Present an alert controller with a text field
Return an array of index paths when searching for a game on the data source
(supports game being on multiple sections, e.g. 'recently played')

Will resolve the part of renaming on #173, still need a way to pick images.

### CODE REVIEW

File: Provenance/Game Library/PVGameLibraryViewController.m [hunk 1]
```diff
@@ -596,31 +596,22 @@ - (void)finishedDownloadingArtworkForURL:(NSString *)url
     }
 }
 
-- (NSIndexPath *)indexPathForGameWithMD5Hash:(NSString *)md5Hash
+- (NSArray *)indexPathsForGameWithMD5Hash:(NSString *)md5Hash
```
Comment (leolobato) [inline Provenance/Game Library/PVGameLibraryViewController.m]: The same game can show up twice now on the library, once on "recent games" and again on it's own section.

**Comment by jasarien** [comment]
Does this PR depend on #318 ?

**Comment by leolobato** [comment]
Nope, it can be applied by itself. But I just found an issue after renaming -- we need to reload the collection view data in case the game has changed places because of the sorting.

I'll push a fix soon.

**Comment by jasarien** [comment]
alright

**Comment by leolobato** [comment]
OK, ready to merge. It was an issue on both tvOS and iOS. If the game changes position on library after the rename, it would end up duplicated on the list.

**Pull request merged**